akka-projection-testkit/src/main/scala/akka/projection/testkit/javadsl/ProjectionTestKit.scala

```diff
@@ -74,9 +72,12 @@ final class ProjectionTestKit private[akka] (testKit: ActorTestKit) {
    *
    * If the assert function doesn't complete without error within the passed `max` duration the test will fail.
    *
+   * Note: when testing a Projection with this method, the Restart Backoff is disabled.
+   * Any backoff configuration settings from `.conf` file or programmatically added will be overwritten.
+   *
```

Comment:
@patriknw, I added this to warn the users about backoff settings potentially being overwritten. Also added the same not in the documentation.

Reply:
Good